src/scenes/AcceleratorRouter/Voting/Voting.tsx

```diff
@@ -9,43 +9,33 @@ import Page from 'src/components/Page';
 import Card from 'src/components/common/Card';
 import { APP_PATHS } from 'src/constants';
 import { useLocalization } from 'src/providers';
-import { requestProjectVotesGet } from 'src/redux/features/votes/votesAsyncThunks';
-import { selectProjectVotes } from 'src/redux/features/votes/votesSelectors';
-import { useAppDispatch, useAppSelector } from 'src/redux/store';
 import strings from 'src/strings';
-import { PhaseVotes, VoteOption } from 'src/types/Votes';
+import { VoteOption } from 'src/types/Votes';
 import useQuery from 'src/utils/useQuery';
-import useSnackbar from 'src/utils/useSnackbar';
 import useStateLocation, { getLocation } from 'src/utils/useStateLocation';
 import { getUserDisplayName } from 'src/utils/user';
 
 import VoteInfo from './VoteInfo';
+import useFetchVotes from './useFetchVotes';
 
 const Voting = () => {
   const history = useHistory();
   const query = useQuery();
   const location = useStateLocation();
   const { activeLocale } = useLocalization();
-  const dispatch = useAppDispatch();
-  const snackbar = useSnackbar();
   const theme = useTheme();
 
   const pathParams = useParams<{ projectId: string }>();
   const projectId = Number(pathParams.projectId);
   const phase = query.get('phase') || 'phase 1'; // default to phase 1?
 
-  const [phaseVotes, setPhaseVotes] = useState<PhaseVotes>();
-  const votes = useAppSelector((state) => selectProjectVotes(state, projectId));
+  const { status, phaseVotes, projectName } = useFetchVotes({ phase, projectId });
 
   const goToEditVotes = useCallback(() => {
     // keep query state for edit view
     history.push(getLocation(APP_PATHS.ACCELERATOR_VOTING_EDIT.replace(':projectId', `${projectId}`), location));
   }, [history, location, projectId]);
 
-  const goToProjects = useCallback(() => {
-    history.push({ pathname: APP_PATHS.ACCELERATOR_OVERVIEW }); // TODO switch to project management lists page
-  }, [history]);
-
   // construct the bread crumbs back to originating context
   const crumbs: Crumb[] = useMemo(
     () => [
```

Comment:
Need activeLocale here, sorry I didn't catch in the previous PR

Reply:
Oops, thanks!  I call this out in other PRs but miss them in mine.